src/mots/templates/directory.template.rst

```diff
@@ -57,16 +57,23 @@
 {% endif %}
 {% endmacro %}
 
-.. note::
-    To add, remove, or update module information, see the `mots documentation <https://mots.readthedocs.io/en/latest/#adding-a-module>`_.
-
 =======
 Modules
 =======
 
-{{ directory.description + "\n" }}
+--------
+Overview
+--------
+To add, remove, or update module information, see the `mots documentation <https://mots.readthedocs.io/en/latest/#adding-a-module>`_.
+
+{{ directory.description }}
+
+
+-----------
+Definitions
+-----------
 
-{%- for module in directory.modules -%}
+{% for module in directory.modules -%}
```

Comment:
Intentional change?

Reply:
Yes, otherwise the output starts on the previous line. This was not needed before because there was a different output that was rendered here.